packages/template-typescript-minimal/app/typescript-file.ts

```diff
@@ -0,0 +1,7 @@
+// Define a variable of type string
+let message: string = 'Hello, TypeScript!'
+
+// Print the message to the console
+console.log(message)
+
+export default message
```

Comment:
This is similar to other boilerplate "delete me" example code in `template-typescript-minimal`. We should build each template in nightly CI runs as a sanity check, in which case this would fail that CI run given the prior (bug) state, and pass post-fix with this file included